examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -7,8 +7,10 @@ import isEqual from "lodash.isequal";
 import React, { useEffect, useState } from "react";
 
 import { externalDataServicePort } from "../mock-external-data-service-interface";
-import type { IAppModel, TaskData } from "../model-interface";
+import type { IAppModel, ITaskData } from "../model-interface";
 
+// Hardcoding a taskListId here for now
```

Comment:
```suggestion
// Hardcoding a taskListId here for now. In a real scenario this would be provided by the user when creating a task list component in the container.
```